store/migrations/migration1551895034/old/models.go

```diff
@@ -0,0 +1,13 @@
+package old
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/chainlink/store/models"
+)
+
+// IndexableBlockNumber captured in old state for tests.
+type IndexableBlockNumber struct {
```

Comment:
Perhaps this should be private so people don't accidentally use it?

Reply:
It's accessed outside the `old` package so needs to be public.

Reply:
It's only used in `store/migrations/migrate_test.go` perhaps it should go there as private?